content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -2,44 +2,112 @@
+   (Packages setting must be enabled for that). Putaway rules are read sequetially until a match is
+   found.
+
+Take the following example:
+
+- If I receive water (category All/drinks), whatever the package, it will be redirected to
+  WH/Stock/Shelf 2/Small Refrigerator
+- If I receive orange juice cans in a box, it will be redirected to WH/Stock/Shelf 2
+- If I receive water or apple juice in a box, it will be redirected to WH/Stock/Shelf 3
+- If I receive a Pallet of Lemonade cans, it will be redirected to WH/Stock/pallets/PAL1
 
-.. image:: putaway/putaw2.png
+.. image:: putaway/putaway-example.png
    :align: center
+   :alt: Some examples of putaway rules.
+
+========================
+Using Storage Categories
+========================
+
+A **Storage category** is an extra location attribute. It allows to define the quantity of products
+that can be stored in the location and how the location will be selected with putaway rules.
+
+Configuration
+==============
 
-Now, if I purchase apples and carrots to my supplier, they will be grouped in the same receipt but redirected to the right location automatically, thanks to putaway rules. This information is available from *Inventory Report*, under the reporting menu.
+In the *Inventory* app, go to :menuselection:`Configuration --> Settings` and activate the *Storage
+Categories*. By doing so, the *Storage Locations* will be automatically activated.
 
-.. image:: putaway/putaw3.png
+Create a Storage Category
+=========================
+
+In the *Inventory* app go to :menuselection:`Configuration --> Storage Categories`. Hit *Create*.
+
+.. image:: putaway/storage-category.png
    :align: center
+   :alt: Create Storage Categories inside Odoo Inventory configuration settings.
+
+Fill a name. You can limit the capacity by weight, by product or package type. The *Allow New
+Product* define when the location is considered available to store a product:
+
+- If location is empty: a product can be added there only if the location is empty
+- If products are the same: a product can be added there only if the same product is already there
+- Allow mixed products: several different products can be stored in this location at the same time
+
+Now we can link a storage category to a location.
 
-.. image:: putaway/putaw4.png
+.. image:: putaway/location-storage-category.png
    :align: center
+   :alt: When a Storage Category is created, it can be linked to a warehouse location.
+
+Storage Categories in putaway rules
+===================================
+
+Lets continue our example from above we applied our 'High Frequency Pallets' on locations PAL1 and
+PAL2 and reworked the putaway rules as follows.
+
+Assume I receive 1 pallet of lemonade cans:
+
+- if PAL1 and PAL2 are empty, the pallet will be redirected to Stock/pallets/PAL1
+- if PAL1 is full, the pallet will be redirected to stock/pallets/PAL2
+- if PAL1 and 2 are full, the pallet will be redirected to stock/Pallets
```

Comment:
```suggestion
Assume receive one pallet of lemonade cans is received:

- If PAL1 and PAL2 are empty, the pallet will be redirected to WH/Stock/Pallets/PAL1.
- If PAL1 is full, the pallet will be redirected to WH/Stock/Pallets/PAL2.
- If PAL1 and 2 are full, the pallet will be redirected to WH/Stock/Pallets.
```

Reply:
removed first `receive` in line 105, kept the 2nd.